packages/mobile/src/config/routing/HomeNavigation.tsx

```diff
@@ -22,5 +25,7 @@ export const HomeStackScreen = () => (
       <HomeStack.Screen name={NavigationName.FILTERSSCREEN} component={FiltersScreen} />
       <HomeStack.Screen name={NavigationName.MAINSCREENHEADER} component={MainScreenHeader} />
       <HomeStack.Screen name={NavigationName.SQUAREOFMOVIE} component={SquareToRenderTemp} />
+      <HomeStack.Screen name={NavigationName.SHORTCUTTOFILTERS} component={ShortcutToFilters} />
```

Comment:
będziesz musiał to inaczej podpiąć, bo główna teraz nawigacja to drawer navigation

Reply:
dodatkowo podepnij to w mniejsce gdzie faktycznie ma być wywoływany ten widok